src/tendril.rs

```diff
@@ -1162,6 +1180,22 @@ where
             }
         }
     }
+
+    unsafe fn raw_len(&self) -> u32 {
+        (*self.buf.get()).heap.len
+    }
+
+    unsafe fn set_len(&mut self, len: u32) {
+        (*self.buf.get()).heap.len = len;
+    }
+
+    unsafe fn aux(&self) -> u32 {
+        (*self.buf.get()).heap.aux
+    }
+
+    unsafe fn set_aux(&self, aux: u32) {
```

Comment:
Should this be `&mut self` like set_len?

Reply:
No because it's used by `make_buf_shared` which is used by public `unsafe_subtendril`. This isn't a public method anyway.